docs/customization.md

```diff
@@ -40,7 +40,63 @@ In json files, you can just override the primary keys you need. You have to over
 
 - `header.json` to define logo URL, default and available languages, number items to flatpages to display in navbar (see default values in https://github.com/GeotrekCE/Geotrek-rando-v3/blob/main/frontend/config/header.json)
 
-- `home.json` to define homepage settings. You can define blocks to display and trek suggestion block with treks ID, outdoor sites ID, services ID or events ID to highlight on homepage (see https://github.com/GeotrekCE/Geotrek-rando-v3/blob/main/frontend/customization/config/home.json).
+- `home.json` to define homepage settings.
+
+  - `suggestions`: You can define blocks to display suggestions groups with treks ID, outdoor sites ID, services ID or events ID to highlight on homepage (see https://github.com/GeotrekCE/Geotrek-rando-v3/blob/main/frontend/customization/config/home.json).
+    Each group has the following properties :
+    ```typescript
+    {
+      "titleTranslationId": string, // you can use locales keys with the files inside `translations` folder
+      "iconUrl": string, // url to the icon file
+      "ids": string[] // list of ids ,
+      "type": 'trek' | 'service' | 'outdoor' | 'events' // if not set, default to `trek`
+    }
+    ```
+    There are two ways to displays suggestions groups:
+    - By using an `object` with the languages code as keys (feature available since 3.8.7 version). By this way you can optimize the valorization of a territory according to the selected language. If you don't need this feature (or if you want the same configuration for several language), use `default` key instead of a language code. The configuration in the example below displays 2 groups of suggestions for all languages except the English version with one different:
```

Comment:
"By this way you can optimize the valorization..." à remplacer par "By this way you can differentiate the valorization..."
C'est pas forcément de l'optimisation... Mais uniquement de différencier les suggestions en fonction de la langue.

Reply:
Done 👍